File: BeatPlannerAndroid/src/beatplanner/ui/views/UpperMeterSpinner.cs

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Android.App;
using Android.Content;
using Android.OS;
using Android.Runtime;
using Android.Util;
using Android.Views;
using Android.Widget;

namespace BeatPlannerAndroid
{
  public class UpperMeterSpinner : MeterSpinner
  {
    public UpperMeterSpinner(Context context) :
      base(context)
    {
    }

    public UpperMeterSpinner(Context context, int mode) :
      base(context, mode)
    {
    }

    public UpperMeterSpinner(Context context, IAttributeSet attrs) :
      base(context, attrs)
    {
    }

    public UpperMeterSpinner(Context context, IAttributeSet attrs, int defStyle) :
      base(context, attrs, defStyle)
    {

    }

    public UpperMeterSpinner(Context context, IAttributeSet attrs, int defStyle, int mode) :
      base(context, attrs, defStyle, mode)
    {

    }

    protected override int TextArrayId
    {
      get
      {
        return Resource.Array.upperMeterValues;
      }
    }
  }
}

File: app/src/main/java/dk/aschoen/beatplanner/ui/views/UpperMeterSpinner.java
//
// Translated by CS2J (http://www.cs2j.com): 2014-07-29 22:02:12
//

package dk.aschoen.beatplanner.ui.views;


import android.content.Context;
import android.util.AttributeSet;

import dk.aschoen.beatplanner.R;

public class UpperMeterSpinner extends MeterSpinner 
{
    public UpperMeterSpinner(Context context) {
        super(context);
    }

    public UpperMeterSpinner(Context context, int mode) {
        super(context, mode);
    }

    public UpperMeterSpinner(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public UpperMeterSpinner(Context context, AttributeSet attrs, int defStyle) {
        super(context, attrs, defStyle);
    }

    public UpperMeterSpinner(Context context, AttributeSet attrs, int defStyle, int mode) {
        super(context, attrs, defStyle, mode);
    }

    protected int getTextArrayId() {
        return R.array.upperMeterValues;
    }

}


